juntagrico/admin.py

```diff
@@ -31,26 +28,29 @@
 from juntagrico.entity.share import Share
 from juntagrico.entity.subs import Subscription, ExtraSubBillingPeriod
 from juntagrico.entity.subtypes import SubscriptionSize, SubscriptionType, SubscriptionProduct
+from juntagrico.util import addons
+
+autodiscover_modules('juntagricoapp', register_to=addons.config)
```

Comment:
Maybe should be called in apps.py instead like in django-admin (indirectly):
 https://github.com/django/django/blob/53209f78302a639032afabf5326d28d4ddd9d03c/django/contrib/admin/apps.py#L24

Reply:
this would indeed be nicer but at the time the admin classes are registered we need the infos about inlines juntagrico apps would like to register
so we have basically two options
we load the modules when the admin is loaded or e load the stuff in the apps.py but juntagrico has to come first in the INSTALLED_APPS setting
since djago tries its best that the order in the INSTALLED_APPS setting is not important I would like to keep it that way and loadd the modules in the admin.py